operations/node_management/NCN_NIC_Replacement.md

```diff
@@ -0,0 +1,120 @@
+# NCN NIC Replacement
+
+This procedure is for readding a non-compute node (NCN) after NIC replacement. This is similar to the
+[Replace NCN procedure](Add_Remove_Replace_NCNs/Add_Remove_Replace_NCNs.md#replace-ncn-procedure) but many of
+the steps can be skipped because only the NIC and its MAC addresses changed.
+
+## Prerequisites
+
+The system is fully installed and has transitioned off of the LiveCD.
+
+All activities required for site maintenance are complete.
+
+The latest CSM documentation has been installed on the master nodes. See
+[Check for Latest Documentation](../../update_product_stream/README.md#check-for-latest-documentation).
+
+## Procedure
+
+### Collect the New MAC Addresses
+
+This step of the NIC replacement procedure describes how to collect the required MAC addresses of the
+interfaces of the new NCN NIC.
+
+The new MAC addresses may be included on a sticker on the NIC card itself. If the new MAC addresses are
+already known, continue to [Edit BSS Boot Parameters](#edit-bss-boot-parameters).
+
+For more information about which MAC address belongs to each named interface, see
+[NCN networking for more information](../../background/ncn_networking.md)
+
+1. Follow the [Collect NCN MAC Addresses](Add_Remove_Replace_NCNs/Collect_NCN_MAC_Addresses.md) procedure.
+
+### Edit BSS Boot Parameters
+
+1. Identify the MAC addresses in BSS that need to be replaced. These can be retrieved from the existing BSS
+boot parameters for the NCN. The MAC addresses that don't match those collected in the previous step are the
+ones that need to be updated. Take note of the MAC addresses being replaced they will be needed in the
+[Clean Up HSM](#clean-up-hsm) step.
+
+    ```screen
```

Comment:
Command blocks in this document do not follow the CSM docs conventions: https://github.com/Cray-HPE/docs-csm/blob/52512e13c1c623c9b8ca4df5a91c899eca866f49/introduction/documentation_conventions.md#command-prompt-reference